ci/publish.rs

```diff
@@ -0,0 +1,406 @@
+//! Helper script to publish the warg suites of crates
+//!
+//! * `./publish bump` - bump crate versions in-tree
+//! * `./publish verify` - verify crates can be published to crates.io
+//! * `./publish publish` - actually publish crates to crates.io
+
+use std::collections::HashMap;
+use std::env;
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::{Command, Stdio};
+use std::thread;
+use std::time::Duration;
+
+// note that this list must be topologically sorted by dependencies
+const CRATES_TO_PUBLISH: &[&str] = &[
+    "cargo-component-macro",
+    "cargo-component-bindings",
+    "cargo-component-core",
+    //"wit",
```

Comment:
The `wit` tool is currently not published with the script as I need to talk to Dan to see if I can yoink his squatted crate name.